tools/cloud/send.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+
+<tool id="send_to_cloud" name="Send to cloud" version="0.1.0" workflow_compatible="false">
+    <requirements>
+        <requirement type="package" version="0.3.3">cloudbridge</requirement>
```

Comment:
@VJalili This requirement is presently listed in the "Admin" -> "Manage dependencies" interface as missing, but is not installable because there is no conda package for `cloudbridge`.
Since this is not actually needed (cloudbridge comes from the Galaxy virtualenv since this tool is listed in `GALAXY_LIB_TOOLS_UNVERSIONED`), I'd suggest to remove or comment out (`<!-- ... -->`) the requirement.

Reply:
@nsoranzo agree, please check https://github.com/galaxyproject/galaxy/pull/7727